app/models/backend_api.rb

```diff
@@ -5,7 +5,8 @@ class BackendApi < ApplicationRecord
   include SystemName
   include ProxyConfigAffectingChanges::ModelExtension
 
-  audited :allow_mass_assignment => true
```

Comment:
why isn't `allow_mass_assignment: true` needed anymore?

Reply:
not present in the audited code anymore, no idea what it did previously